proposals/extensions.md

```diff
@@ -632,41 +855,62 @@ The preceding rules mean:
 - and that extension methods declared directly in a namespace take precedence
   over extension methods imported into that same namespace with a using namespace directive.
 
-TODO(static) clarify behavior for extension on `object` or `dynamic` used as `dynamic.M()`?
-
-### Indexer access
+#### Indexer access
 
-TL;DR: If no candidate is applicable, then we attempt extension indexer access instead.
+TL;DR: For non-extension types, we'll fall back to an implicit extension member lookup. For extension types, we include indexers from the underlying type.
 
 We modify the [element access rules](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#128111-general) as follows:
 
-/[...]
 An *element_access* is dynamically bound if \[...]
 If the *primary_no_array_creation_expression* of an *element_access* is a value of an *array_type*, the *element_access* is an array access. 
-Otherwise, the *primary_no_array_creation_expression* shall be a variable or value of a class, struct, or interface type  TODOTODO
+Otherwise, the *primary_no_array_creation_expression* shall be a variable or value of a class, struct, interface, /***or extension** type
```

Comment:
Did you mean to write `\*` instead of `/*` ? But I don't know why would that be needed, this seems to work fine:

```suggestion
Otherwise, the *primary_no_array_creation_expression* shall be a variable or value of a class, struct, interface, **or extension** type
```

Applies to other similar changes throughout the doc - I see usages of `/*` and `/[` that were probably meant to be `\*` and `\[` (although the escaping might be also unnecessary).

Reply:
Tried again and it looks like the escape sequence is no longer needed. Removed :-)